internal/fake/sdk.go

```diff
@@ -46,17 +46,17 @@ func NewSDKEvaluator(
 	}
 }
 
-func (s SDKEvaluator) EvaluateRequestPolicy(req *http.Request, userInfo types.User, permission *openapi.RondConfig) (core.PolicyResult, error) {
+func (s SDKEvaluator) EvaluateRequestPolicy(ctx context.Context, req core.RondInput, userInfo types.User) (core.PolicyResult, error) {
```

Comment:
is core.RondInput the actual input? I'd rename the function parameter accordingly